lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/LinearModelResult.cxx

```diff
@@ -269,15 +309,21 @@ void LinearModelResult::load(Advocate & adv)
   MetaModelResult::load(adv);
   adv.loadAttribute( "basis_", basis_ );
   adv.loadAttribute( "design_", design_ );
-  adv.loadAttribute( "beta_", beta_ );
+  if (adv.hasAttribute("beta_"))
+    adv.loadAttribute("beta_", coefficients_);
+  else
+    adv.loadAttribute( "coefficients_", coefficients_ );
```

Comment:
I would switch the order: test if the object has the correct attribute first, then fall back to the old way if it does not. This way it is consistent with the way you do it for residualtsVariance_ / sigma2_

Reply:
```suggestion
  if (adv.hasAttribute("coefficients_"))
    adv.loadAttribute( "coefficients_", coefficients_ );
  else
    adv.loadAttribute("beta_", coefficients_);
```